unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -431,6 +431,10 @@ Show [\u20b9]
 # [\p{Alphabetic}] ∥ \p{Script=Common}
 #  & [\p{Decomposition_Type=None} \p{Decomposition_Type=Canonical}]
 
+# Nonalphabetic bindus
```

Comment:
```suggestion
# The UTC 172 script ad hoc report (L2/22-128) item VII 27 “Nonalphabetic bindus” points out that
# “Most characters with InSC=Bindu have Alphabetic=Yes.” but there could be exceptions.
# TODO: Review with SAH:
# - is the rule roughly right?
# - if so, are these intentional exceptions?
# - otherwise, are these wrong? should these 5 be alphabetic?
```

Reply:
Done in https://github.com/unicode-org/unicodetools/pull/318/commits/e610d63ab47717ed45dd59f652e474f6f833a90e